INSTALL.md

```diff
@@ -176,3 +184,16 @@ Start up the rails server by running the following command:
 $ UNICORN_WORKERS=2 rails server -b 0.0.0.0 # Runs the server, available at localhost:3000
 ```
 > The number of unicorn workers is variable and sometimes it will need more than 2. In case the server is slow or start suffering from timeouts, try restarting porta with a higher number like 8.
+
+Set the domain for the Master, Provider & Developer portals via the rails console. Whichever hostname being used for the local machine should be the value set for each attribute (the following domain names are just examples and updating the */etc/hosts* file is also required):
```

Comment:
This is not necessary at all. It creates the domains by default. It is still ok to do it and mention it somewhere if you want, but this is not part of the installation so it shouldn't be in `INSTALL.md`, and anyway it is important to say clearly that this step is **optional**.

Reply:
ah okay yes it should be optional, it creates the domains but they cannot be used, they are just example.com, is there another README where this step would be more appropriate?

Reply:
@kevprice83 What do you mean that they can't be used? 😱 They should be able to be used. I can use them:

### Tenant
![image](https://user-images.githubusercontent.com/11318903/57387370-7c01f380-71b6-11e9-96ed-ca36d5529eb9.png)

### Master
![image](https://user-images.githubusercontent.com/11318903/57387358-760c1280-71b6-11e9-88a2-94f87b525ca8.png)

Reply:
Well the accounts were created in my case with the following domains: **master-admin.example.com**, **provider-admin.example.com**, **provider.example.com** so these were obviously never going to work unless they are added to the _/etc/hosts_. Did you pass a param when starting up the rails server to add **.lvh.me**? @Martouta

Reply:
The *.lvh.me*  resolves itself and all its subdomains to `127.0.0.1`. So you don't need to setup your own DNS or change your `/etc/hosts`.

Reply:
You can change the `example.com` anyway passing an ENV `THREESCALE_SUPERDOMAIN` or making a `config/settings.yml` with this with the default you want:
https://github.com/3scale/porta/blob/04e65efabe4311ad75a70b18074e48c4d4d49101/config/examples/settings.yml#L2
And the `.lvh.me` is because we don't want to resolve to real URLs in development and we used to use `.dev` but we had problems with it for an update of Chrome so now we use this one, as you can customise as well if you want to (but only if you know well what you are doing):
https://github.com/3scale/porta/blob/04e65efabe4311ad75a70b18074e48c4d4d49101/config/examples/settings.yml#L41

Reply:
Understood @Martouta @didierofrivia I'll make the change, I didn't know this.

Reply:
As long as you have internet or a DNS server setup correctly with `.lvh.me` pointing to `127.0.0.1`

But I think that is out of topic for the README, we can do a Pro-tip
What would be good is to begin to have wiki page for those pro-tips instead of files.

Reply:
Yeah I just wasn't aware that `lvh.me` was a standard DNS entry mapped back to `127.0.0.1` I think this could be fine in another wiki or README I suppose.